pilot/pkg/model/authentication.go

```diff
@@ -69,6 +69,11 @@ func ConstructSdsSecretConfig(serviceAccount string, sdsUdsPath string) *auth.Sd
 								GoogleGrpc: &core.GrpcService_GoogleGrpc{
 									TargetUri:  sdsUdsPath,
 									StatPrefix: SDSStatPrefix,
+									ChannelCredentials: &core.GrpcService_GoogleGrpc_ChannelCredentials{
+										CredentialSpecifier: &core.GrpcService_GoogleGrpc_ChannelCredentials_LocalCredentials{
+											LocalCredentials: &core.GrpcService_GoogleGrpc_GoogleLocalCredentials{},
```

Comment:
and what is local credential and channel credential?

Reply:
details can be found https://grpc.io/docs/guides/auth.html